kpconv_torch/train.py

```diff
@@ -293,8 +292,5 @@ def train(
     # Training
     trainer.train(net, training_loader, test_loader, config)
 
```

Comment:
What was the initial purpose of these lines? Why exiting the current process?

Reply:
That's an excellent question! I do not know...

Killing the Python program at this stage is a really strange way of doing, the final print is never executed (why do not let the program finish by itself?). Plus it makes the functional test fail: if the process is killed, the `pytest` session can't finish.

As I do not see any good reason of keeping it, I prefer to remove this killing statement.